sanic/response.py

```diff
@@ -118,12 +118,7 @@ def get_headers(
         )
 
         headers = self._parse_headers()
-
-        if self.status == 200:
-            status = b"OK"
-        else:
-            status = STATUS_CODES.get(self.status)
-
+        status = STATUS_CODES.get(self.status, b"UNKNOWN RESPONSE")
```

Comment:
I thought this was added already 🤔

Reply:
@ahopkins yeah, but it was merged after 19.12.2 went out. It merges the fix into the branch that I cut from 19.12.2

Reply:
got it 👍